src/interfaces/IAuthOptions.spec.ts

```diff
@@ -98,6 +98,16 @@ describe('IAuthOptions', () => {
       ).toBe(true)
     })
 
+    it('accepts data that has algorithm, a string secretOrPublicKey and a boolean authorizationRequired', () => {
+      expect(
+        isAuthOptions({
+          algorithm: EncryptionAlgorithms.ES256,
+          authorizationRequired: true,
```

Comment:
let's please call the new option `credentialsRequired` to have the same naming scheme as `express-jwt`, the library this is inspired by.

Reply:
done